blogpost/Blogpost-draft.md

```diff
@@ -1,39 +1,268 @@
 
+#### Freestyle Jobs vs. Pipelines
+<a name=freestyleVsPipelines/> 
+
+The conventional way of building applications with Jenkins is to define a series of sequential executable steps, each configured as a freestyle job, and then trigger one
+after the other until the whole build process is completed. The chaining of freestyle jobs is usually done via the Jenkins UI, which makes changes and reconfigurations of 
+the job chain rather inconvenient. But fear not, the Jenkins Pipeline plugin is here for the rescue! 
+With the new plugin it becomes possible to orchestrate all workflow steps from commit to deployment within a single job. 
+This job is defined using the Pipeline DSL (domain specific language), which is based on Groovy. 
+Let's have a quick look at the vocabulary of the Pipeline DSL: in the following we have a listing of the three most important terms when it comes to writing pipeline scripts.
+
+* **Stage**: stages are used to break your pipeline into logically distinct sections, which can include one or more build steps. All steps defined within a stage are then visualized as a unique segment in the Pipeline UI.
+* **Step**: build steps are the single executable tasks, which can be chained together in sequence to form stages.
+* **Node**: nodes are wrappers for resource-intensive steps, which should be scheduled and executed in a separate workspace, in order to avoid negative impacts on the overall pipeline performance.
+
+So using a single job definition for creating build chains already makes life more convenient, but there is still more to it than that. The Pipeline plugin also offers new features that could not be realized so easily prior to Jenkins 2.0:
+
+* Interruptions of the pipeline process (e.g. by restart of the Jenkins master) can be handled gracefully now with a persistent record of execution.
+* Manual quality gates can be introduced into the workflow, where the pipeline process halts and waits for manual input.
+* More complex workflows can be designed which include forks, joins, loops and parallel executions.
+
+The following picture is a screenshot of the Jenkins Pipeline UI of our projects, showing the pipeline currently waiting for manual input at the quality gate.
+
+![](images/pipeline.png)
+
+
+#### Pipeline as Code
+<a name=pipelineAsCode/> 
+
+Another big advantage of the Pipeline plugin is the 'Pipeline as code' feature. Instead of defining a Pipeline job via the Jenkins UI, 
+it is now possible to store the configuration in an external file (called Jenkinsfile), which can easily be put under version control. 
+Jenkins can then be configured to automatically scan repositories for Jenkinsfiles and setup a new build pipeline accordingly. 
+Having the project code as well as the build/deployment configuration under the same roof seems to be the new trend at the moment, 
+but after some discussion on this topic, we still remain skeptical towards this approach (see [conclusion](#conclusion)). 
```

Comment:
Können wir diesen und den vorherigen Satz in den nächsten Absatz über Multibranch-Pipelines packen? Ich finde das an dieser Stelle total verwirrend - vor allem, weil nicht klar wird, ob unsere Skepsis dem Multibranch-Ansatz oder generell für 'Pipeline as code' gilt.